packages/core/src/IRenderer.ts

```diff
@@ -111,7 +111,7 @@ export interface IRendererRenderOptions
  * Starard Interface for a Pixi renderer.
  * @memberof PIXI
  */
-export interface IRenderer extends SystemManager, GlobalMixins.IRenderer
+export interface IRenderer<VIEW = ICanvas> extends SystemManager, GlobalMixins.IRenderer
```

Comment:
Same with other `VIEW = ICanvas` that should be `VIEW extends ICanvas = ICanvas`